<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtToolTestApproximatorXyzMainDialog</class>
 <widget class="QDialog" name="QtToolTestApproximatorXyzMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X (key)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_int_x">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y (value)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="box_double_y">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Z (key)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_int_z">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button">
       <property name="text">
        <string>Add tuple</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
